src/tools/create-app-tester.js

```diff
@@ -36,7 +36,7 @@ const promisifyHandler = handler => {
 };
 
 // A shorthand compatible wrapper for testing.
-const createAppTester = appRaw => {
+const createAppTester = (appRaw, storeKey = null) => {
```

Comment:
this is good, but typically it's good to avoid `null`. It behaves weirdly with default args. 

That said, you don't actually have to set a default here. The verbose version of this function signature is `(appRaw, storeKey=undefined)`, but since js doesn't really enforce arguments at all, it's most clearly written as `(appRaw, storeKey)`. Does that make sense?

Reply:
I see now that you likely copied the `null` from below. Honestly I shouldn't have written it like that in the first place. 😁